src/containers/Board/Board.js

```diff
@@ -187,14 +193,28 @@ export class Board extends Component {
     }));
   };
 
+  handleBoardKeyUp = (event) => {
+    if (event.keyCode === 27) {
```

Comment:
I would refactor the key code literal into an enum reference like:
```
const keyCodes = {
  ESCAPE: 27
}
```
There's probably an NPM package for this, either way should be a key-codes.js file that will be imported

Reply:
added [keycode] npm package and refactored above :)